platform/darwin/src/MGLLogging_Private.h

```diff
@@ -0,0 +1,10 @@
+#import "MGLLogging.h"
+
+@interface MGLLogging (Private)
+
+- (void)info:(NSString *)fileName message:(id)message, ...;
+- (void)debug:(NSString *)fileName message:(id)message, ...;
```

Comment:
These method names don’t conform to Cocoa naming conventions. For one thing, since this method’s first selector piece is a verb, it sounds like it has the effect of debugging something. Moreover, `fileName` is not something that gets debugged. Consider a more descriptive, if verbose, name such as `-logDebugMessage:atPath:`.

Reply:
This API is meant to be private because it will serve only as a tool to communicate what is going on in the SDK. I didn't consider any cocoa convention because my intention was to specify the action first as you call the class `[MGLLogging sharedInstance] debug:message:]` which seemed clear. I will try other naming as variadic parameters must be at the end.